examples-staging/ecommerce/keystone.ts

```diff
@@ -46,18 +46,16 @@ export default withAuth(
         credentials: true,
       },
     },
-    db: process.env.DATABASE_URL
-      ? { provider: 'postgresql', url: process.env.DATABASE_URL }
-      : {
-          provider: 'sqlite',
-          url: databaseURL,
-          async onConnect(context) {
-            console.log('Connected to the database!');
-            if (process.argv.includes('--seed-data')) {
-              await insertSeedData(context);
-            }
-          },
-        },
```

Comment:
We don't support changing which database provider you're using dynamically like this, and doing this messed up running because on some jobs in CI, we have a `DATABASE_URL` env variable set.